client/inbox-panel/index.js

```diff
@@ -53,8 +60,9 @@ const renderNotes = ( {
 	isBatchUpdating,
 	lastRead,
 	notes,
-	onDismiss,
+	dismissNote,
 	onNoteActionClick,
+	setShowDismissAllModal,
```

Comment:
Minor and semantic, but maybe we could make this prop `onDismissAll` to keep parent logic separate?

Reply:
Instead of changing the original function name to `onDismissAll`, I aliased it in the `renderNotes` in https://github.com/woocommerce/woocommerce-admin/pull/7871/commits/51f3ddf94839fd84cc21426ec45bc7e0c58a7b6b. What do you think?